### DISCUSSION TITLE
Project Brutality Weapons

### DISCUSSION THREAD

**Discussion by AvrasMitilineos** [community/community#137396] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello! According to https://github.com/pa1nki113r/Project_Brutality/wiki/_Arsenal-and-Equipment the Black Hole Generator of Project Brutality can be summoned by using the Console Command "Summon BHGen". Now, this command can't be used on the Latest Version of Project Brutality. Does anybody know how can I summon the Black Hole Generator?

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by AvrasMitilineos** [comment]
Hello! I would like to ask you if there is a way to be notified if a New Version of Project Brutality Staging on https://github.com/pa1nki113r/Project_Brutality/tree/PB_Staging in order to download it. Thanks!
